tox.ini

```diff
@@ -7,7 +7,7 @@ skipsdist = True
 usedevelop = True
 setenv =
    VIRTUAL_ENV={envdir}
-   PYTHONWARNINGS=default::DeprecationWarning
+   PYTHONWARNINGS=default::DeprecationWarning,ignore::DeprecationWarning:distutils,ignore::DeprecationWarning:site,ignore:Using or importing the ABCs,ignore:dns.hash module,ignore:Please provide `is_available
```

Comment:
What's with the suppressed warnings?  Do we need to fix something up?

Reply:
No, they're all from modules deep down below our code.